### PR TITLE
Fix incorrect resolution date on SKIP1

### PR DESCRIPTION
**Pull request by jni**
## Description

In the rush to merge #4182 before @stefanv's bedtime, I committed the wrong date to the resolution tag for SKIP1 - governance. This fixes that mistake.

### CODE REVIEW

**Comment by jni** [comment]
Note: I incorrectly changed in #4182 the index page to 2x5 instead of 5x2. The second commit fixes that.

**Comment by jni** [comment]
Note some more: for some reason, the docs build fine on my machine, which had sphinx 1.8 but now has 2.2, builds fine on both, but not on Travis. Thankfully, the build error was pretty clear, so the next build should pass... :crossed_fingers:

**Comment by jni** [comment]
Looking good! CC @scikit-image/core would be good to get this merged ASAP as @stefanv's link on the mailing list won't work until this is in master!

File: doc/source/index.rst [hunk 1]
```diff
@@ -39,23 +39,23 @@ Sections
...
 
        How decisions are made in scikit-image.
 
-     - `Installation Steps <install.html>`_
+   * - `Installation Steps <install.html>`_
 
        How to install scikit-image.
 
-   * - `User Guide <user_guide.html>`_
+     - `User Guide <user_guide.html>`_
```
Comment (soupault) [inline doc/source/index.rst]: Wait, why no asterisk here? 🤔

Comment (jni) [reply doc/source/index.rst]: @soupault there should be an asterisk every 2nd heading. Asterisk is the start of a new row.

Comment (jni) [reply doc/source/index.rst]: I only just discovered this myself, hence this element of the PR. :joy:

**Comment by jni** [comment]
@soupault thanks for the approval! :pray:

**Pull request merged**